src/UI/WB.UI.Designer/Controllers/Api/Headquarters/HQQuestionnareBackupController.cs

```diff
@@ -47,7 +47,7 @@ public IActionResult Verify(Guid questionnaireId)
                 return this.ErrorWithReasonPhraseForHQ(StatusCodes.Status404NotFound, string.Format(ErrorMessages.TemplateNotFound, questionnaireId));
             }
 
-            if (!this.ValidateAccessPermissions(questionnaireView))
+            if (!(this.ValidateAccessPermissions(questionnaireView) || User.IsAdmin()))
```

Comment:
Could you please move this check for admin inside the method ValidateAccessPermissions?
Either overload it or add a new one to support admin or allow admin user.

Reply:
added new method